mirascope/base/calls.py

```diff
@@ -109,3 +110,42 @@ def _setup(
             tool_types = kwargs.pop("tools")
             kwargs["tools"] = [tool_type.tool_schema() for tool_type in tool_types]
         return kwargs, tool_types
+
+
+BaseModelT = TypeVar("BaseModelT", bound=BaseModel)
+BaseCallT = TypeVar("BaseCallT", bound=BaseCall)
+
+
+def create_call(
+    base_model: type[BaseModelT],
```

Comment:
Shouldn’t this be a BaseCallT?

Reply:
For this and the same change below make sure to update the docstrings accordingly

Reply:
Actually this should just be BaseCall